src/main/kotlin/no/nav/familie/dokument/storage/StonadController.kt

```diff
@@ -34,54 +34,32 @@ class StonadController(@Autowired val storage: MellomLagerService,
         validerGyldigJson(søknad)
         val directory = contextHolder.hentFnr()
 
-        try {
-            storage.put(directory, stønad.stønadKey, søknad)
-        } catch (e: RuntimeException) {
-            secureLogger.warn("Kunne ikke mellomlagre overgangsstønad for $directory", e)
-            return ResponseEntity.status(HttpStatus.INTERNAL_SERVER_ERROR).build()
-        }
-
+        storage.put(directory, stønad.stønadKey, søknad)
         return ResponseEntity.status(HttpStatus.CREATED).build()
     }
 
     @GetMapping(path = ["/{stonad}"], produces = [MediaType.APPLICATION_JSON_VALUE])
     fun hentMellomlagretSøknad(@PathVariable("stonad") stønad: StønadParameter): ResponseEntity<String> {
         val directory = contextHolder.hentFnr()
-        return try {
-            val data = storage[directory, stønad.stønadKey]
-            ResponseEntity.ok(data)
-        } catch (e: RuntimeException) {
-            if (e is AmazonS3Exception && e.statusCode == 404) {
-                ResponseEntity.noContent().build()
-            } else {
-                secureLogger.info("Noe gikk galt", e)
-                ResponseEntity.status(HttpStatus.INTERNAL_SERVER_ERROR).build()
-            }
-        }
+        return ResponseEntity.ok(storage[directory, stønad.stønadKey])
```

Comment:
This now throws 404 instead of 204, does this effect frontend anything? 
It will happen a lot that it throws 404. So we should maybe add catch here for 404 and return ResponseEntity.notFound, instead of logging it